src/nose_html_reporting/__init__.py

```diff
@@ -189,6 +189,7 @@ def report(self, stream):
         self.report_file.write(self.jinja.get_template(os.path.basename(self.report_template_filename)).render(
             report=self.report_data,
             stats=self.stats,
+            timestamp=datetime.now().strftime("%Y/%m/%d %H:%M:%S"),
```

Comment:
Due to it is not our code, please add a comments before this line in order to explain what do you want.